scripts/g.extension/g.extension.py

```diff
@@ -2218,7 +2251,7 @@ def resolve_known_host_service(url, name, branch):
         else:
             actual_start = ""
         if "branch" in match["url_end"]:
-            suffix = match["url_end"].format(name=name, branch=branch)
+            suffix = match["url_end"].format(name=name, branch=get_default_branch(url))
```

Comment:
This is what seems to me like always using default, never user-provided branch name.

Reply:
yes, right. Thanks for catching this.